src/portable/ver.rs

```diff
@@ -50,7 +50,8 @@ pub enum FilterMinor {
 }
 
 static BUILD: Lazy<Regex> = Lazy::new(|| {
-    Regex::new(r#"^\d+\.\d+(?:-(?:alpha|beta|rc|dev)\.\d+)?\+(?:[a-f0-9]{7}|local)$"#).unwrap()
+    Regex::new(r#"^\d+\.\d+(?:\.\d+)?(?:-(?:alpha|beta|rc|dev)\.\d+)?\+(?:[a-f0-9]{7}|local)$"#)
+        .unwrap()
```

Comment:
The change here was to support the third number for patch versions? `(?:\.\d+)?`
Because extensions do publish patch releases?

Reply:
Our extension(s) have a third version number -- so we need to parse this out. Long term it would be nice to use a more structured form for this rather than strings and regexps, but I didn't want to duplicate the whole PackageInfo struct tree.